File: tests/fixtures/generic/clean/taint_rce_clean.cs
// @audit-fixture
// @rule: taint_rce
// @category: security
// @expected: clean
using System.Diagnostics;
public class CommandController : Controller {
    private static readonly string[] ALLOWED = { "report", "status" };
    public IActionResult Run() {
        var cmd = Request.Query["cmd"].ToString();
        if (!ALLOWED.Contains(cmd)) return Forbid();
        Process.Start(cmd);
        return Ok();
    }
}

File: tests/fixtures/generic/clean/taint_rce_clean.py
# @audit-fixture
# @rule: taint_rce
# @category: security
# @expected: clean
import subprocess, shlex
from flask import request
@app.route("/run")
def run():
    cmd = shlex.quote(request.args.get("cmd"))
    subprocess.call(["echo", cmd])
